### PR TITLE
Environment lists

### PR DESCRIPTION
**Pull request by drummond-dds**
Paired with @dandds to extract out some styles for the accordion table from @montana-mil's PR and make them generic so they can be reused across the app.

### CODE REVIEW

File: atst/routes/portfolios/applications.py [hunk 1]
```diff
@@ -57,23 +57,28 @@ def create_application(portfolio_id):
         )
 
 
+def environments_obj(application):
```
Comment (dandds) [inline atst/routes/portfolios/applications.py]: This is a little Ruby-ish. I think a more Pythonic name would be something like `environment_data_for_app`.

Comment (montana-mil) [reply atst/routes/portfolios/applications.py]: what makes it ruby/what makes this name pythonic?

Comment (dandds) [reply atst/routes/portfolios/applications.py]: Ruby methods tend to have names that are just nouns, like `repo.pull_requests`. Python doesn't put quite as much emphasis on OO and, importantly, we're not in a class context here. Python functions tend to have names that describe what they do, so even `get_environment_obj_for_app` might be better.

Comment (montana-mil) [reply atst/routes/portfolios/applications.py]: oh i see, i didnt even realize it was a function. the new name definitely helps!

**Pull request merged**